frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -897,7 +909,7 @@ function VirtualKeyboard:setLayer(key)
         self.umlautmode = not self.umlautmode
     end
     self:initLayer()
-    self:_refresh(true)
+    self:_refresh(false)
```

Comment:
Also nope, as the flash is mildly useful there on eInk, and can be disabled in the eInk settings ;).